detections/network/internal_vertical_port_scan.yml

```
@@ -1,9 +1,9 @@
 name: Internal Vertical Port Scan
 id: 40d2dc41-9bbf-421a-a34b-8611271a6770
-version: 7
-date: '2025-08-18'
-author: Dean Luxton
-status: production
+version: 8
+date: '2025-09-18'
+author: Dean Luxton,Splunk, troyheady2, Github Community
```

The issue highlighted a suggestion and does not qualify to be an author as this will claim this whole rule was written by a GitHub community contributor.

I suggest you credit them in the release notes (which is more than enough).

```suggestion
author: Dean Luxton, Splunk
```